composer.lock

```diff
@@ -475,16 +475,16 @@
         },
         {
             "name": "psr/log",
-            "version": "1.1.2",
```

Comment:
Let's remove this whole file from the commit since it's not necessary for your PR.

Reply:
Fixed on recent push.